the-collab-lab/tcl-72-smart-shopping-list
# Ho list page

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._

## Description

This code change/update updates the List Page UI and provides route protection to relevant pages

## Related Issue
closes #32 

## Acceptance Criteria

 - Remove "Hello from the /list page!"
 - Make everything centered
 - Import checkbox, button, and search input components from MUI
 - Whenever an item is checked, the item is crossed out on the UI, and the purchase urgency is removed for 24 hrs
 - Space out the item and the delete button
 - Change the delete button to an icon
 - Add route protection for relevant pages
 - Show selected list across all pages

## Type of Changes
`Bug fix`
`UI enhancement`

## Updates

### Before
![Screenshot from 2024-03-27 14-29-59](https://github.com/the-collab-lab/tcl-72-smart-shopping-list/assets/89491787/9e4f4eda-41d0-46c9-934c-bf9bd7664734)

### After
![Screenshot from 2024-03-27 14-30-19](https://github.com/the-collab-lab/tcl-72-smart-shopping-list/assets/89491787/dd8318e8-0bda-428c-b153-75771f06776b)

## Testing Steps / QA Criteria

Do a git pull and checkout to ho-listPage branch. Run npm start to start development server. Navigate to list page to see UI updates. Also sign out of the app and try to route to the list and mange-list page. You shouldn't be able to access the pages if not signed in
